pkg/node/utils.go

```diff
@@ -192,7 +196,9 @@ func findMounts(drives []*direct_csi.DirectCSIDrive) error {
 	for scanner.Scan() {
 		line := scanner.Text()
 		words := strings.SplitN(line, " ", 2)
-		index[words[0]] = line
+		if _, ok := index[words[0]]; !ok {
```

Comment:
choose the first line (earliest mount) for a particular block device. This is there in case the same drive is bind mounted at multiple locations